src/apps/backend/modules/account/internal/store/account_model.py

```diff
@@ -1,34 +1,57 @@
 from datetime import datetime
-from typing import Annotated, Any, Optional
-
+from typing import Any, Optional
 from bson import ObjectId
-from pydantic import BaseModel, ConfigDict, Field
-from pydantic.functional_validators import AfterValidator
-
 from modules.account.types import PhoneNumber
 from modules.object_id.utils import object_id_validate
-
-PyObjectId = Annotated[ObjectId | str, AfterValidator(object_id_validate)]
-
-
-class AccountModel(BaseModel):
-    model_config = ConfigDict(arbitrary_types_allowed=True)
-
-    id: Optional[PyObjectId] = Field(None, alias="_id")
-    active: bool = True
-    first_name: str = ""
-    hashed_password: str = ""
-    phone_number: Optional[PhoneNumber] = None
-    last_name: str = ""
-    username: str = ""
-    created_at: Optional[datetime] = datetime.now()
-    updated_at: Optional[datetime] = datetime.now()
+import json
+
+class AccountModel:
+    def __init__(
+        self,
+        id: Optional[ObjectId | str] = None,
```

Comment:
Avoid using default values. Engineer often tend to make mistakes without fully understanding the implications of the defaults.